backend/tournesol/views/video.py

```diff
@@ -154,13 +162,19 @@ def get_queryset(self):
             for crit in settings.CRITERIAS
         ]
         criteria_weight = Case(*criteria_cases, default=0)
-        queryset = (
-            queryset.annotate(
-                total_score=Sum(F("criteria_scores__score") * criteria_weight)
-            )
-            .filter(total_score__gt=0)
-            .order_by("-total_score")
+
+        queryset = queryset.annotate(
+            total_score=Sum(F("criteria_scores__score") * criteria_weight)
         )
+
+        if show_unsafe is True:
+            queryset = queryset.order_by("-total_score")
```

Comment:
Now that no minimal number of contributions is required, this queryset would also include videos without any rating (e.g videos that has been added to a rate-later list), and with score being null. So an additional filter is necessary:
```suggestion
            queryset = (
                queryset
                .filter(total_score__isnull=False)
                .order_by("-total_score")
            )
```

Reply:
Done